config/jobs/kubernetes/sig-release/release-branch-jobs/1.17.yaml

```diff
@@ -1170,7 +1170,13 @@ presubmits:
           value: external-dependencies-version vendor vendor-licenses
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200802-ff98b58-1.17
         name: main
-        resources: {}
+        resources:
+          limits:
+            cpu : "2"
```

Comment:
```suggestion
            cpu: "2"
```

Reply:
Also, did you have a specific reason for using `1.5Gi` instead of recommended `1.2Gi`?

Reply:
It was a mistake. It's fixed. :-)